src/transformers/testing_utils.py

```diff
@@ -2554,6 +2581,20 @@ def wrapper(*args, **kwargs):
     return decorator
 
 
+def run_first(test_case):
+    """
+    Decorator marking a test with order(1). When pytest-order plugin is installed, tests marked with this decorator
+    are garanteed to run first.
+
+    This is especially useful in some test settings like on a Gaudi instance where a Gaudi device can only be used by a
+    single process at a time. So we make sure all tests that run in a subprocess are launched first, to avoid device
+    allocation conflicts.
+    """
+    import pytest
+
+    return pytest.mark.order(1)(test_case)
```

Comment:
cc @ydshieh looks nice for us as well!

Reply:
yeah it might help reduce flakiness when testing order is important.
the plugin is a [pytest-dev official](http://github.com/pytest-dev/pytest-order), and supports wide range of inter-tests dependency [features](https://github.com/pytest-dev/pytest-order?tab=readme-ov-file#features).